plugins/module_utils/network/common/network_template.py

```diff
@@ -29,6 +29,9 @@
 
 __metaclass__ = type
 
+# moved actual class in `rm_base.network_template`
+# this is kept here for backwards compatibility
+# will be removed in ansible.netcommon version 3.0.0
```

Comment:
```suggestion
# TODO: Remove after 2023-01-01
```
This is more preference than the other one, but being able to `git grep TODO` and get all the pending removals makes removing them later much easier